.pre-commit-config.yaml

```diff
@@ -112,3 +112,16 @@ repos:
         language: system
         pass_filenames: false
         always_run: true
+
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.5
+    hooks:
+      - id: codespell
+        name: codespell
+        description: Checks for common misspellings in text files.
+        entry: codespell --toml=api/pyproject.toml
+        language: python
+        types: [text]
+        additional_dependencies:
+          - tomli
```

Comment:
```suggestion
```

I belive this dependency is redundant, as the TOML library is available in python 3.11+

Reply:
According to your [`pyproject.toml` file the minimum supported Python version is `3.10`](https://github.com/equinor/template-fastapi-react/blob/5ee221123b0d262b4b54c36a7ef9382fa6b83281/api/pyproject.toml#L10), so that is why I included it.